packages/lesswrong/components/common/Header.jsx

```diff
@@ -48,6 +48,8 @@ const styles = theme => ({
     verticalAlign: 'middle',
     fontSize: 19,
     position: "relative",
+    display: 'flex',
```

Comment:
Another dumb question / I guess I just want to completely grok all the css we put in: What does this do?